www/js/survey/multilabel/MultiLabelButtonGroup.tsx

```diff
@@ -100,11 +100,12 @@ const MultilabelButtonGroup = ({ trip, buttonsInline=false }) => {
           )
         })}
       </View>
-      <View>
-        <IconButton icon='check-bold' mode='outlined' size={16} onPress={verifyTrip}
-                    disabled={trip.verifiability != 'can-verify'}
-                    style={{width: 20, height: 20, margin: 3}}/>
-      </View>
+      {trip.verifiability === 'can-verify' && (
+        <View style={{marginTop:'1rem'}}>
```

Comment:
```suggestion
        <View style={{marginTop: 16}}>
```
`rem` units are great for web apps but they don't actually exist in the React Native spec. If you saw a warning from the linter in VSCode, that is why.

Using `rem` does happen to work right now because we are using React Native Web, but our eventual goal is React Native (not web). So we should try to avoid these web-specific units as we move forward.

Reply:
Ah, good to know!  Interesting it let me get away with using 'rem' at all - my thought was to use the rem to help show it was offsetting the text - but since it doesn't work, I'll stick with the straight numbers!